src/main/java/org/opensearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -23,14 +23,15 @@ public class PluginSettings {
 
     private static PluginSettings instance;
     public static final String CONFIG_FILES_PATH = "config/";
-    private static final String DEFAULT_CONFIG_FILE_PATH =
+    public static final String DEFAULT_CONFIG_FOLDER_PATH =
             OPENSEARCH_HOME
                     + File.separator
                     + "config"
                     + File.separator
                     + "opensearch-performance-analyzer"
-                    + File.separator
-                    + "performance-analyzer.properties";
+                    + File.separator;
+    private static final String DEFAULT_CONFIG_FILE_PATH =
+            DEFAULT_CONFIG_FOLDER_PATH + "performance-analyzer.properties";
```

Comment:
Are we still using `OPENSEARCH_HOME` here for the default config folder?

Reply:
Using ```OPENSEARCH_HOME``` here for the default config folder. Removing the default maybe a better idea. Any thoughts? 

`performance-analyzer-agent` which starts the PA process will never succeed if the setting is not present, which means this DEFAULT value will never be used.

Reply:
I am ok to remove the var if never used.